### PR TITLE
Update issue templates to the template folder

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <yb532204897@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes

Issues templates should be put to the `ISSUE_TEMPLATE` directory so that they can be selected when u open an issue.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-name: "Feature Request"
+name: "\U0001F680 Feature Request"
```
Comment (bwplotka) [inline .github/ISSUE_TEMPLATE/feature-request.md:2]: `\U0001F680`
What's this?

Comment (yeya24) [reply .github/ISSUE_TEMPLATE/feature-request.md:2]: :bug:

Comment (yeya24) [reply .github/ISSUE_TEMPLATE/feature-request.md:2]: :rocket: We can remove this emoji if you think it is not necessary

**Review by kakkoyun — APPROVED** [review]
👍 LGTM

**Comment by yeya24** [comment]
Is it ready to go?

**Comment by kakkoyun** [comment]
I leave the decision to @bwplotka since he has risen some concerns about 🚀 😆

**Review by bwplotka — APPROVED** [review]

**Pull request merged**